src/ThermalScattering.tex

```diff
@@ -125,7 +125,32 @@ \subsubsection{\textsf{ITXE} Block}\label{sec:ITXEBlock}
 
 Because the use of discrete outgoing energies and cosines can result in unphysical spikes in the neutron flux spectrum at thermal energies, some Monte Carlo codes attempt to ``smear'' the outgoing energies and cosines to produce a smoother distribution (that more closely approximates a continuous distribution with $\nxs{7}=2$).
 
-\todo[inline]{Describe the format for $\nxs{7}=2$.}
+When $\nxs{7}=2$, the distribution of outgoing energies for each incident energy is continuous in energy and specified by a probability density function and cumulative distribution function. The format of the \Block{ITXE} in this case is given in \Tableref{tab:ITXEBlockContinuousHeader} and \Tableref{tab:ITXEBlockContinuousData}. As before, the index at the start of the block is \startblock{ITXE}=\jxs{3}. Unlike in the $\nxs{7} < 2$ cases, the number of outgoing energies for each incident energy is allowed to vary. The number of discrete cosines, $N_\mu=$\nxs{3}-1, remains the same for each pair of incident and outgoing energies, however.
+
+\begin{XSSTable}{Start of \Block{ITXE} for $\nxs{7}=2$}
+  \startblock{ITXE}          & \var{L}$(l),l=1,\ldots,N_{in}$  & Location in \XSS\ of distribution for incident energy $l$ \\
+  \startblock{ITXE}+$N_{in}$ & $N_E(l),l=1,\ldots,N_{in}$      & Number of outgoing energies for incident energy $l$
+  \label{tab:ITXEBlockContinuousHeader}
+\end{XSSTable}
+
+\begin{XSSTable}{\Block{ITXE} for $\nxs{7}=2$ (continued)}
+  \var{L}(1)+1                     & $E_1^{out}[E_{in}(1)]$                     & First of \var{NIEB}\tnotex{tn:nieb}\ outgoing energies for inelastic scattering at $E_{in}(1)$    \\
+  \var{L}(1)+2                     & $\mathrm{PDF}_1[E_{in}(1)]$          & Probability density function value for $E_1^{out}[E_{in}(1)]$ \\
+  \var{L}(1)+3                     & $\mathrm{CDF}_1[E_{in}(1)]$          & Cumulative distribution function value for $E_1^{out}[E_{in}(1)]$ \\
+  \var{L}(1)+4                     & $\mu_l(1\rightarrow 1), l=1,\ldots,N_\mu$  & Discrete cosines for scattering from $E_{in}(1)$ to $E_1^{out}[E_{in}(1)]$    \\
+  \var{L}(1)+4+$N_\mu$             & $E_2^{out}[E_{in}(1)]$                     & Second of \var{NIEB} outgoing energies for inelastic scattering at $E_{in}(1)$    \\
```

Comment:
The index on this line (and subsequent lines) needs to be incremented. These need to be `\var{L}(2)` instead of `\var{L}(1)`.

Reply:
It is actually L(1). L(2) refers to the start of the data corresponding to the second _incident energy_.This line here refers to the start of the data corresponding to the second _outgoing energy_ for the first incident energy.